index.html

```diff
@@ -1,56 +1,67 @@
-  <title>Hacktoberfest 2020!!!</title>
-  <!-- JQUERY -->
-  <script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js'></script>
-  <!-- Bootstrap -->
-  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
-  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" crossorigin="anonymous"></script>
-  <!-- CSS -->
-  <link rel="stylesheet" href="style.css">
-
-</head>
-
-<body>
-  <div class="container-fluid">
-    <div class="row nav">
-      <div class="col-md-12">
-        <span class="nav-text">Hacktoberfest 2020</span>
-      </div>
-    </div>
-    <div>
-    <!-- Github Usernames -->
-    <div class="row mt-3">
-      <div class="col-md-12">
-        <img src="./git.png" height="20px">
-        <span class="git-header">Github Detail</span>
-      </div>
-    </div>
-    <div class="d-flex flex-column">
-      <ul>
-      <li><a href="https://github.com/anku580" target="blank">Anku</a></li>
-      <li><a href="https://github.com/MohammedAkhil" target="blank">Mohammed Akhil</a></li>
-    </ul>
-    </div>
-    <!-- Linkedin Usernames -->
-    <div class="row mt-3">
-      <div class="col-md-12">
-        <img src="./linkedin.jpg" height="20px">
-        <span class="linkedin-header">Linkedin Detail</span>
-      </div>
-    </div>
-    <div class="d-flex flex-column">
-      <ul>
-      <li><a href="https://www.linkedin.com/in/anku580/" target="blank">Anku</a></li>
-      <li><a href="https://www.linkedin.com/in/s-mohammed-akhil/" target="blank">Mohammed Akhil</a></li>
-    </ul>
-    </div>
-  </div>
-  </div>
-</body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Hacktoberfest 2020!!!</title>
+  <!-- JQUERY -->
+  <script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js'></script>
+  <!-- Bootstrap -->
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" crossorigin="anonymous"></script>
+  <!-- CSS -->
+  <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+  <div class="container-fluid">
+    <div class="row nav">
+      <div class="col-md-12">
+        <span class="nav-text">Hacktoberfest 2020</span>
+      </div>
+    </div>
+    <div>
+    <!-- Github Usernames -->
+    <div class="row mt-3">
+      <div class="col-md-12">
+        <img src="./git.png" height="20px">
+        <span class="git-header">Github Detail</span>
+      </div>
+    </div>
+    <div class="d-flex flex-column">
+      <ul>
+      <li><a href="https://github.com/anku580" target="blank">Anku</a></li>
+
+	   <li><a href="https://github.com/mendelevium28" target="blank">Muskan deep Kaur</a></li>
```

Comment:
Hi Muskan, can you please remove the extra spaces and add the name in the order wise. For example. `Mohammed Akhil` name was there, can you please add your after that. 

This will help in keeping the file in a good shape. You can amend the existing commit instead of creating a new one. Let me know if you need any support on how to amend the commit. 

You can use this https://www.freeformatter.com/html-formatter.html website to format the code.